### ISSUE TITLE
Shape measurements, add aspect ratio description

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#666]
The example figure contains the "aspect ratio" measurement. I could be nice to add this to the explained shape features in Activity 1. 

Generally speaking, maybe the general explanations of shape features in Activity 1 should be moved to the "Explanations" section.

**Comment by manerotoni** [comment]
Yes, better in explanation and may be with some illustrations. I would not explain all of the shape measurements. Only those you want to highlight in the activity.
